hbase_storage/pom.xml

```diff
@@ -27,6 +27,33 @@
             <artifactId>pipeline_api</artifactId>
             <version>2.7.0.1808-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-network-common_${scala.binary.version}</artifactId>
```

Comment:
Yes since it is in the platform specific build.  However, since these artifacts are dynamic there is no dependency logic carried forward.  If you add hbase_storage as a dependency, that jar is all you get.